src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let cost = 0;
+
+  if (days < 3) {
+    return days * 40;
+  }
+
+  if (days < 7) {
+    cost = (days * 40) - 20;
+
+    return cost;
+  }
+
+  if (days >= 7) {
```

Comment:
Just a suggestion: I think you can use `else if` instead of putting spaces between if statements. It is both less lines and easier to read in my opinion.

Reply:
multiple else if are definitely not easy to read